specification/refarch.adoc

```diff
@@ -428,11 +428,13 @@ and G-stage paging hardware, the root security manager (TSM-driver) must use MTT
 to isolate supervisor domain memory. In this deployment model,
 TEE and TVM address spaces are identified by supervisor domain identifiers
 (Smsdid) to maintain the isolation during access and in internal
-address translation caches, e.g. Hart TLB lookup may be extended with the
+address translation caches, e.g., Hart TLB lookup may be extended with the
 SDID in addition to the ASID, VMID for workloads in the Confidential supervisor
 domain. TVM memory isolation must support sparse memory management
 models and architectural page-sizes of 4KB, 64K, 2MB, 1GB (and optionally
```

Comment:
Should 64K be 64KB? Is there RISC-V MMU spec for 64KB pages?

Reply:
yes 64KB - RISC-V priv ISA with Svnapot supports 64KB pages